scripts/bb-test-zfstests.sh

```diff
@@ -52,16 +52,9 @@ $ZFSTESTS $TEST_ZFSTESTS_OPTIONS \
     -d $TEST_ZFSTESTS_DIR \
     -s $TEST_ZFSTESTS_DISKSIZE \
     -r $TEST_ZFSTESTS_RUNFILE &
-CHILD=$!
-
-sleep 1
-TEST_LOG=$(ls -t /var/tmp/test_results/*/log | head -1)
-rm -f log
-ln -s $TEST_LOG log
-
-wait $CHILD
```

Comment:
Oh I overlooked that when reviewing zfsonlinux/zfs#5293, that's not really what we want.  We want ed the symlink to be created in the `/var/tmp/test_results/` directory next to the dated directories.  This has the advantage of always being writable and predictable.